### ISSUE TITLE
Meta description not show

### ISSUE THREAD

**Issue by wiwatsrt** [flarum/core#1057]
Meta description not show
## Bug report
- Version of Flarum v0.1.0-beta.6
## Additional comments

I set description in admin but not show in page
## Code

```
<meta name="description" content="">
```

**Comment by Jurgyte** [comment]
Same here. It seems that in Flarum "flarum/core/src/Http/WebApp/WebAppView.php" is small section:

        $view->title = $this->buildTitle(array_get($forum, 'data.attributes.title'));
        $view->description = $this->description;

Description is not set by anything. Could someone check?

**Comment by daggeto** [comment]
👍  and how to customize that description will be same as first post in page?

**Comment by Luceos** [comment]
There is no setter method for description. This is confirmed, fix should take place somewhere in the mentioned `WebAppView` class:

https://github.com/flarum/core/blob/master/src/Http/WebApp/WebAppView.php#L286

**Comment by franzliedke** [comment]
I can't test this right now, but I believe changing that line to the following should fix the issue:

~~~php
$view->description = array_get($forum, 'data.attributes.description');
~~~

(The `$description` attribute can then also be removed.)

**Comment by franzliedke** [comment]
Pretty sure this was fixed in #1105.
